larray/core/axis.py

```diff
@@ -3186,7 +3186,7 @@ def combine_axes(self, axes=None, sep='_', wildcard=False, front_if_spread=False
                     combined_labels = _axes[0].labels
                 else:
                     sepjoin = sep.join
-                    axes_labels = [np.array(label, np.str, copy=False) for label in _axes.labels]
+                    axes_labels = [np.array(label, str, copy=False) for label in _axes.labels]
```

Comment:
This change on your part is correct (it is equivalent to the old deprecated code) but this smells like an error in the old code. I always thought np.str was numpy's native string type 😢. I will have to investigate this.

Reply:
Here is the deprecation warning from numpy 1.20:
```
DeprecationWarning: `np.str` is a deprecated alias for the builtin `str`. To silence this warning, use `str` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.str_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```